app/src/main/java/com/github/wnder/picture/InternalCachePictureDatabase.java

```diff
@@ -202,4 +182,14 @@ public void updateLocalScoreboard(String uniqueId, Map<String, Double> scoreboar
     public void deleteLocalPicture(String uniqueId){
         localDatabase.deletePicture(uniqueId);
     }
+
+    /**
+     * To ret when an online-only feature is accessed offline
+     * @return
```

Comment:
You forgot the return statement in the javadoc

Reply:
done